mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -1,18 +1,114 @@
 @import "src/styles/variables";
 
+$maxWidth: 1200px;
```

Comment:
why 1200?

Reply:
What max-width container do we need? Changed in 1370px like  figma maket

Reply:
the problem is with consistency. We have global scss variables for such purposes

Reply:
added to global variables